core/modules/macros/jsontiddlers.js

```diff
@@ -20,14 +20,19 @@ exports.name = "jsontiddlers";
 
 exports.params = [
 	{name: "filter"},
-	{name: "spaces"}
+	{name: "spaces"},
+	{name: "escapeUnsafeScriptCharacters"}
 ];
 
 /*
 Run the macro
 */
-exports.run = function(filter,spaces) {
-	return this.wiki.getTiddlersAsJson(filter,$tw.utils.parseInt(spaces));
+exports.run = function(filter,spaces,escapeUnsafeScriptCharacters) {
+	var json = this.wiki.getTiddlersAsJson(filter,{
+		spaces: $tw.utils.parseInt(spaces),
```

Comment:
With this `.parseInt(space)` ... you hardcode "spaces" into the JSON output. I think it would be better to allow `\t` here. See description below. The spec would allow `\t`.
According to the macro docs, the default is 4 spaces (which I can't see?). ... While 1 `\t` would do the same thing. Saving 3 characters. 

>JSON.stringify()
>
>space Optional
    A **_String_** or **_Number_** object that's used to insert white space into the output JSON string for readability purposes.
>
>    If this is a Number, it indicates the number of space characters to use as white space; this number is capped at 10 (if it is greater, the value is just 10). Values less than 1 indicate that no space should be used.
>
>    If this is a String, the string (or the first 10 characters of the string, if it's longer than that) is used as white space. If this parameter is not provided (or is null), no white space is used.
>
> from: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

In line 706 of `core/modules/wiki.js` I think, the code is more flexible. IMO would allow us to use a `\t` tab.

Reply:
Good point @pmario, but the handling of the spaces parameter is unchanged in this PR, and so I think the point might be better made in a separate issue.

> According to the macro docs, the default is 4 spaces

I think that is wrong, the default is not to use whitespace.

Reply:
I mean I think *the documentation* is wrong.